packages/flutter/test/material/expansion_panel_test.dart

```diff
@@ -1298,8 +1298,57 @@ void main() {
 
     // Padding is added to expanded header
     box = tester.renderObject(find.ancestor(of: find.byKey(firstPanelKey), matching: find.byType(AnimatedContainer)).first);
-    expect(box.size.height, equals(80.0)); // _kPanelHeaderCollapsedHeight + 32.0 (double default padding)
-    expect(box.size.width, equals(736.0));
+    expect(box.size.height, equals(80.0)); // _kPanelHeaderCollapsedHeight + 24.0 (double default padding)
+    expect(box.size.width, equals(744.0));
+  });
+
+  // Regression test for #5848
```

Comment:
Nit: This is usually formatted with the full URL like this:

```
// Regression test for https://github.com/flutter/flutter/issues/5848
```

Reply:
Thanks, I should have seen that in the other test cases!